virttest/utils_test/libvirt.py

```diff
@@ -2616,13 +2623,18 @@ def do_migration(vm_name, uri, extra, auth_pwd, auth_user="root",
             else:
                 logging.error("The real prompt text: <%s>", text)
                 break
-
+        log = session.get_output()
         session.close()
+        if return_ret_output is True:
+            return (True, log)
         return True
```

Comment:
@waynesun09 , ok. I searched tp-libvirt, qemu and tp-spice, and no more use of this function. So I will change the return directly.